recipes/shapelib/bld.bat

```diff
@@ -0,0 +1,18 @@
+:: Make a build folder and change to it.
+mkdir build
+cd build
+
+:: Configure using the CMakeFiles
+%LIBRARY_BIN%\cmake -G "NMake Makefiles" -DCMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" -DCMAKE_BUILD_TYPE:STRING=Release ..
```

Comment:
I like to break lines with the `^` and use the env var `%SRC_DIR%` instead of `..`. But that is style and you do not need to change if you disagree.